app/presenters/two_factor_authentication/backup_code_selection_presenter.rb

```diff
@@ -8,5 +8,28 @@ def method
     def security_level
       I18n.t('two_factor_authentication.two_factor_choice_options.least_secure_label')
     end
+
+    def disabled?
+      !user.nil? && user.backup_code_configurations.any?
+    end
+
+    def mfa_configuration
+      return '' if !disabled?
+      if user.backup_code_configurations.unused.count == 1
+        return t(
+          'two_factor_authentication.two_factor_choice_options.unused_backup_code',
+          count: mfa_configuration_count,
+        )
+      else
+        return t(
+          'two_factor_authentication.two_factor_choice_options.unused_backup_code_plural',
+          count: mfa_configuration_count,
+        )
+      end
```

Comment:
1. looks like this top branch is not covered by tests
2. the `return` keyword is not needed
3. We can take advantage of Rails I18n built in pluralization: https://guides.rubyonrails.org/i18n.html#pluralization

So this can all be:
```suggestion
      t(
        'two_factor_authentication.two_factor_choice_options.unused_backup_code',
        count: mfa_configuration_count,
      )
```

If we change the translations to be structured like:
```yml
two_factor_authentication:
  two_factor_choice_options:
    unused_backup_code:
      one: 'one thing'
      other: '%{count} things'
```